CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [3.2.0] - 2020-07-16
+### Added
+- Added `Patches::Worker` extra parameters to support forward compatibility with the upcoming releases
 
-## [3.0.1] - 2018-11-19
+## [3.1.0] - 2018-11-19
```

Comment:
Don't change history 😉

Reply:
If anything a 3.1.0 section needs to be added. And it looks like we never tagged these releases.

Reply:
ah, right, thanks for pointing out. I was confused by the version number 👍